### ISSUE TITLE
Pretty formatting puts commas inside empty arrays or objects

### ISSUE THREAD

**Issue by jkeljo** [dpranke/pyjson5#26]
The stock `json` module doesn't do this. Repro:

```
% python3.7 
Python 3.7.0 (default, Oct  2 2018, 09:20:07) 
[Clang 10.0.0 (clang-1000.11.45.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> import json5
>>> json.dumps([], indent=2)
'[]'
>>> json5.dumps([], indent=2)
'[\n  ,\n]'
>>> json.dumps({}, indent=2)
'{}'
>>> json5.dumps({}, indent=2)
'{\n  ,\n}'
```

**Comment by jkeljo** [comment]
Also looks like these strings won't round-trip through the parser:

```
>>> json5.loads(json5.dumps({}, indent=2))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/json5/lib.py", line 80, in loads
    raise ValueError(err)
ValueError: <string>:2 Unexpected "
" at column 4
```

**Comment by dpranke** [comment]
Thanks for the report. 

The '[\n  ,\n]' is definitely a bug.

The fact that the strings won't round-trip is not a bug, because the input string isn't legal :)

That fact that the error message is reporting the wrong unexpected character, however, is a bug :).

These should be easy fixes, I'll try to get to them today.

**Comment by dpranke** [comment]
Fixed in 6c4038e449e2083f1f09328a1f92cbe702607dea (v0.8.5). This actually required a fix to the underlying parser generator (glop, v0.6.2) to fix the error reporting, so double-bonus bug report :).
